### PR TITLE
Update the repo-create template with more questions

### PR DESCRIPTION
**Pull request by spiffxp**
These are questions I find myself guessing the answer to or
having to ask as followup requests to the initial create

I'm also adding "staging" as an option since we tend to
create a bunch of those and they have slightly different
requirements from new or migrated repos

/cc @cblecker @calebamiles
Since you've also handled repo creation in the past

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/repo-create.md [hunk 1]
```diff
@@ -13,15 +13,36 @@ e.g. cloud-provider-foo
...
+Please prove you have followed the appropriate approval process for this new
+repo by including links to the relevant approvals (meeting minutes, e-mail
+thread, etc.)
+
+Authoritative requirements are here: https://git.k8s.io/community/github-management/kubernetes-repositories.md
+
+tl;dr (but really you should read the linked doc, this may be stale)
+- If this is a core repository, then sig-architecture must approve
+- If this is a SIG repository, then this must follow the procedue spelled out
+in your SIG's charterthe SIG Technical Leads must approve
```
Comment (cblecker) [inline .github/ISSUE_TEMPLATE/repo-create.md]: this doesn't look right

Comment (spiffxp) [reply .github/ISSUE_TEMPLATE/repo-create.md]: good catch, that was intended to be an e.g. which I have decided to drop in favor of "just check the charter"

**Comment by spiffxp** [comment]
@cblecker updated, PTAL

File: .github/ISSUE_TEMPLATE/repo-create.md [hunk 2]
```diff
@@ -13,15 +13,36 @@ e.g. cloud-provider-foo
...
+e.g. chris, dianne
+
+### If a new repo, who should be listed as approvers in OWNERS
+e.g. alice, bob
+
+### If a new repo, who should be listed in SECURITY_CONTACTS
+e.g. alice, bob
+
+### What should the repo description be
+by default we will set it to "created due to (link to this issue)"
```
Comment (cblecker) [inline .github/ISSUE_TEMPLATE/repo-create.md]: I don't like using the link as a default description, but not strongly enough to actually hold this up.

**Pull request merged**